policy/authorization/psp.rego

```diff
@@ -0,0 +1,15 @@
+package authorization
+
+deny[{
+	"id": "podsecuritypolicies-kube-system",
```

Comment:
How will this policy be 'audit' ?

Reply:
I don't think the 'audit' use case makes sense for the 'authorization' scenario. The usual scenario is that you check with 'audit' if existing resources match to the policies. The authorization calls are only partly create/update on resources and the majority are like get/use/port-forward/exec/logs/... requests and are not leaving any persistent resources in Kubernetes. For example there is no way to write on rule which applies at the same time to:
* The request from Kubernetes to use a specific PodSecurityPolicy (from the kube-controller-manager)
* The then created Pod which now has this PodSecurityPolicy

The /authorization call matches on SubjectAccessReviews from the APIServer, but these SubjectAccessReviews are afaik not persistently stored in Kubernetes and are just transient resources.

Reply:
The scenario is as follows 
An `admin` persona user is adding a new policy to the system and wants to `audit` the affect of the policy before making a change. So in theory he can apply policy on current state of the cluster (based on past requests)

Reply:
I think that absolutly makes sense for Validating/MutatingWebhooks. I think for the Authorization Use Case that won't work because they match on the SubjectAccessReview and not on the Resources itself (like they to in the Validating/MutatingWebhooks)